source/getting-started/installation-raspberry-pi-all-in-one.markdown

```diff
@@ -56,6 +56,7 @@ To upgrade the All-In-One setup manually:
 *  Change to hass user `sudo su -s /bin/bash hass`
 *  Change to virtual enviroment `source /srv/hass/hass_venv/bin/activate`
 *  Update HA `pip3 install --upgrade homeassistant`
+*  Type `exit` to get out of hass user and change to normal user.
```

Comment:
Any work on this @AdithyanI? You can edit your current pull request at https://github.com/AdithyanI/home-assistant.github.io/blob/patch-1/source/getting-started/installation-raspberry-pi-all-in-one.markdown